lib/findingaids/ead/behaviors.rb

```diff
@@ -64,13 +64,26 @@ def ead_date_display(parts = Array.new)
     return parts.join(" ")
   end
 
-  # Returns the language terms as string from a given three-letter code found in language_map.properties
-  def get_language_from_code code, properties = Hash.new
-    file = File.new(File.path(Rails.root + "config/translation_maps/language_map.properties"))
```

Comment:
@eshadatta Remove this file too if it's not longer being used: config/translation_maps/language_map.properties